### DISCUSSION TITLE
Can't create new user (Unhandled Rejection (Error): GraphQL error: Forbidden)

### DISCUSSION THREAD

**Discussion by jakhongir-i** [strapi/strapi#6798] [Legacy Topics]
When I try to create a new user, the server returns an error 

![IMAGE 2020-06-24 21:43:24](https://user-images.githubusercontent.com/46269246/85617695-1d792c80-b679-11ea-8472-3ba7fc8adfb6.jpg)
![IMAGE 2020-06-24 21:43:25](https://user-images.githubusercontent.com/46269246/85617708-21a54a00-b679-11ea-8c24-6323e1f20a8a.jpg)
![IMAGE 2020-06-25 00:19:10](https://user-images.githubusercontent.com/46269246/85617964-852f7780-b679-11ea-9cc8-fb729b638e20.jpg)

and if send a request in postman, then everything is fine

![IMAGE 2020-06-25 00:17:55](https://user-images.githubusercontent.com/46269246/85617844-57e2c980-b679-11ea-9555-47ba185caddd.jpg)

this is how I make the mutation 

![IMAGE 2020-06-24 21:43:21](https://user-images.githubusercontent.com/46269246/85618108-b9a33380-b679-11ea-9570-1da695953538.jpg)
![IMAGE 2020-06-25 00:22:00](https://user-images.githubusercontent.com/46269246/85618235-e6efe180-b679-11ea-89e4-58e0456a507a.jpg)

**Comment by PrateekSonkar** [comment]
@jakhongir-i  did you find the solution, I am facing this issue too?

**Reply by jakhongir-i** [reply]
find! 
try :

```
import gql from "graphql-tag";

const SIGNUP_MUTATION = gql`
  mutation CreateUser($username: String!, $email: String!, $password: String!) {
    register(
      input: { username: $username, email: $email, password: $password }
    ) {
      jwt
      user {
        username
        email
      }
    }
  }
`;

export default SIGNUP_MUTATION;
```

**Reply by PrateekSonkar** [reply]
@jakhongir-i  thank you very much 👍 !!
